jest/setup.js

```diff
@@ -34,3 +34,11 @@ NativeModules.AndroidBpkSnackbar = {
   show: jest.fn(),
   dismiss: jest.fn(),
 };
+
+NativeModules.BPKSnackbarManager = {
+  LENGTH_SHORT: 0,
```

Comment:
What is this?

Reply:
To mock these out during tests. It means we exercise all the code in `BpkSnackbar.js` and `NativeSnackbar.ios.js` but not the actual native implementation